apps/cowswap-frontend/src/modules/permit/hooks/usePermitInfo.ts

```diff
@@ -23,6 +23,13 @@ import { IsTokenPermittableResult } from '../types'
 
 const UNSUPPORTED: PermitInfo = { type: 'unsupported', name: 'native' }
 
+export const PERMIT_GAS_LIMIT_MIN: Record<SupportedChainId, number> = {
+  [SupportedChainId.MAINNET]: DEFAULT_MIN_GAS_LIMIT,
+  [SupportedChainId.GNOSIS_CHAIN]: DEFAULT_MIN_GAS_LIMIT,
+  [SupportedChainId.SEPOLIA]: DEFAULT_MIN_GAS_LIMIT,
+  [SupportedChainId.GOERLI]: 36_000,
+}
```

Comment:
Moved this part into the permit module so the lib doesn't need to know about SupportedChainId